templates/mail/repo/release.tmpl

```diff
@@ -50,12 +49,12 @@
 			{{end}}
 		</ul>
 	</p>
-	<div class="footer">
-	<p>
-		---
-		<br>
-		<a href="{{.Link}}">{{.locale.Tr "mail.view_it_on" AppName}}</a>.
-	</p>
+	<div style="font-size:small; color:#666;">
```

Comment:
Since we are using this snippet multiple times, wouldn't it make sense to extract it into its own template?